web/src/styles/base/height.scss

```diff
@@ -0,0 +1,15 @@
+.min-height-40rem {
```

Comment:
If we're breaking this out into it's own stylesheet, my comment about programmatically generating these classes on the other commit would actually be easier to implement.

Reply:
spoke to JP, leaving as is due to the limited number of css class names that will be changed